recipe/build.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+export CPPFLAGS="$CPPFLAGS -I$PREFIX/include"
+
+# Filter out -std=.* from CXXFLAGS as it disrupts checks for C++ language levels.
```

Comment:
Do you know which package sets `-std=.*`. It's not toolchain AFAIK.

Reply:
I got that from defaults but already removed it. It did not help with the issue here.